### ISSUE TITLE
Write separate parsers for mangareader, mangastream and shonen jump official

### ISSUE THREAD

**Issue by pandusonu2** [NITDgpOS/manga#63]
The heading says it all :)

**Comment by shiningsurya** [comment]
`mangastream` is a good site, also they don't keep complete archive of all the manga scalations they release(they ask you to buy manga when they become available in your region); I don't think writing a parser for `mangastream` would be of any benefit since if it's there, leecher sites like mangapanda, mangareader mirror it,

**Comment by pandusonu2** [comment]
Yeah, but mangastream has quicker releases, so if you want to check latest, mangastream is better. We already have the "Chapter not available" in place, so if the user tries to downlaod a previous chapter, it would show the same.

**Comment by shiningsurya** [comment]
They are highly dedicated and motivated to the cause of providing high quality scalation products. I'm not denying the end-user would definitely want to use `mangastream` as source, but, it sort of feels bad. I've disabled my ad block program on their website. 
I leave it to your good judgment.

**Comment by pandusonu2** [comment]
I think I can even add official ShonenJump site for this now as they are releasing everything official. 
Updating the issue name with the changes
